aslprep/interfaces/bids.py

```diff
@@ -12,12 +11,12 @@
     traits,
 )
 from niworkflows.interfaces.bids import DerivativesDataSink as BaseDerivativesDataSink
-from pkg_resources import resource_filename as pkgrf
 
 from aslprep import config
+from aslprep.data import load as load_data
 
 # NOTE: Modified for aslprep's purposes
-aslprep_spec = loads(Path(pkgrf("aslprep", "data/aslprep_bids_config.json")).read_text())
+aslprep_spec = loads(load_data.readable("aslprep_bids_config.json").read_text())
```

Comment:
:+1: I would pretty much only use `readable()` in the form `load_data.readable(...).read_<text|bytes>()`, as otherwise you need to check that whoever you're passing it to is only going to use `read_text()` or `read_bytes()`.